### ISSUE TITLE
What's the meaning "nilboy" in loss function return? And how can I calculate the accuracy? 

### ISSUE THREAD

**Issue by mariannanovaes** [nilboy/tensorflow-yolo#57]
Can I use the nilboy information to calculate the accuracy?

**Comment by zsano1** [comment]
What is the nilboy's purpose? Isn't it just a predict cell coordinate? Or it is just the author's name??

**Comment by zsano1** [comment]
What is the nilboy's purpose? Isn't it just a predict cell coordinate? Or it is just the author's name??
